tests/LSP14Ownable2Step/LSP14Ownable2Step.behaviour.ts

```diff
@@ -609,4 +609,37 @@ export const shouldBehaveLikeLSP14 = (
       });
     });
   });
+
+  describe.only("should be able to have 2-step `renoucneOwnership()` on a fresh blockchain", () => {
+    before(async () => {
+      context = await buildContext();
+
+      // Skip 1 block
+      // Simulate real blockchain
+      // You would never be able to deploy a contract and initiate `renounceOwnership(..)`
```

Comment:
Good test and comments. Just suggested some rewording


```suggestion
      // in a real blockchain environment, you would not likely be able to deploy a contract 
      // and initiate `renounceOwnership(..)` in the block 0
```